test-case/check-pause-resume.sh

```diff
@@ -75,24 +78,25 @@ esac
 [[ -z $file_name ]] && file_name=$dummy_file
 
 func_pipeline_export "$tplg" "type:$test_mode & ${OPT_VALUE_lst['S']}"
-for idx in $(seq 0 $(expr $PIPELINE_COUNT - 1))
+for idx in $(seq 0 $((PIPELINE_COUNT - 1)))
 do
     # set up checkpoint for each iteration
     func_lib_setup_kernel_checkpoint
-    channel=$(func_pipeline_parse_value $idx channel)
-    rate=$(func_pipeline_parse_value $idx rate)
-    fmt=$(func_pipeline_parse_value $idx fmt)
-    dev=$(func_pipeline_parse_value $idx dev)
-    snd=$(func_pipeline_parse_value $idx snd)
-
-    dlogi "Entering expect script with: $cmd -D $dev -r $rate -c $channel -f $fmt -vv -i $file_name -q"
+    channel=$(func_pipeline_parse_value "$idx" channel)
+    rate=$(func_pipeline_parse_value "$idx" rate)
+    fmt=$(func_pipeline_parse_value "$idx" fmt)
+    dev=$(func_pipeline_parse_value "$idx" dev)
+    snd=$(func_pipeline_parse_value "$idx" snd)
 
     # expect is tcl language script
     #   expr rand(): produces random numbers between 0 and 1
     #   after ms: Ms must be an integer giving a time in milliseconds.
     #       The command sleeps for ms milliseconds and then returns.
+    dlogi "Entering expect script with:
```

Comment:
@marc-hb I see we have duplicated logs, example:

https://sof-ci.01.org/softestpr/PR527/build423/devicetest/?model=ICL_RVP_HDA&testcase=check-pause-resume-playback-10
```
2020-11-23 22:32:27 UTC [REMOTE_INFO] Entering expect script with:
      aplay   -D hw:0,5 -r 48000 -c 2 -f S16_LE -vv -i /dev/zero -q
spawn aplay -D hw:0,5 -r 48000 -c 2 -f S16_LE -vv -i /dev/zero -q
```

Reply:
The log was already duplicated, I just added a newline that made it more obvious. Before:

```shell
git checkout origin/master
./test-case/check-pause-resume.sh

2020-12-02 04:25:33 UTC [INFO] Entering expect script with: aplay -D hw:0,0 -r 48000 -c 2 -f S16_LE -vv -i /dev/zero -q
spawn aplay -D hw:0,0 -r 48000 -c 2 -f S16_LE -vv -i /dev/zero -q
```
I can easily escape the newline so the output is exactly like before.

I'm a bit reluctant to trust and rely entirely on `expect` for the logging... maybe that's also why someone else doubled this?

Reply:
Good point here, let's compare the cmd here with both logs.